documentation/src/main/asciidoc/topics/cache_api.adoc

```diff
@@ -199,7 +193,7 @@ These are, in order of cost:
 * writing to a cache store (optional)
 * locking
 
-As of {brandname} 4.0, using the async methods will take the network calls and marshalling off the critical path.  For various technical reasons, writing to a cache store and acquiring locks, however, still happens in the caller's thread.  In future, we plan to take these offline as well.  See link:http://lists.jboss.org/pipermail/infinispan-dev/2010-January/002219.html[this developer mail list thread] about this topic.
+Using the async methods will take the network calls and marshalling off the critical path.  For various technical reasons, writing to a cache store and acquiring locks, however, still happens in the caller's thread.
```

Comment:
The second sentence is no longer valid. @wburns fixed it with the non-blocking work.

Reply:
removed the 2nd sentence. thanks @pruivo